src/main/scala/github/gphat/censorinus/statsd/Encoder.scala

```diff
@@ -7,14 +7,14 @@ import github.gphat.censorinus._
   */
 object Encoder extends MetricEncoder {
 
-  def encode(metric: Metric): String = metric.metricType match {
-    case "c" => encodeCounter(metric)
-    case "g" => encodeGauge(metric)
-    case "h" => encodeTimer(metric) // StatsD doesn't support histograms, use timer instead?
-    case "m" => encodeMeter(metric)
-    case "ms" => encodeTimer(metric)
-    case "s" => encodeSet(metric)
-    case _ => "" // TODO Complain!
+  def encode(metric: Metric): Option[String] = metric.metricType match {
+    case "c" => Some(encodeCounter(metric))
+    case "g" => Some(encodeGauge(metric))
+    case "h" => Some(encodeTimer(metric)) // StatsD doesn't support histograms, use timer instead?
+    case "m" => Some(encodeMeter(metric))
+    case "ms" => Some(encodeTimer(metric))
+    case "s" => Some(encodeSet(metric))
+    case _ => None
```

Comment:
This was just so we could push the complaining into the `Client`, instead of doing it in the encoder.